week-1/errors/2.js

```diff
@@ -1,5 +1,8 @@
 // Currently trying to print the string "I was born in Bolton" but it isn't working...
 // what's the error ?
 
-console.log(`I was born in ${cityOfBirth}`);
+//console.log(`I was born in ${cityOfBirth}`);
+//const cityOfBirth = "Bolton";
+
 const cityOfBirth = "Bolton";
```

Comment:
Good work, could you explain why this works and the previous one didn't work?

Reply:
before console cityOfBirth  I should give value to cityOfBirth so I write const cityOfBirth = "Bolton"; 
so when I call cityOfBirth , my out put is Bolton . 
I hope my explain undrstandeble :)